README.md

```diff
@@ -593,6 +593,18 @@ metrics. Those pushes are still counted as
 the `rate` of this metric, but you have to inspect the logs to identify the
 offending pusher.
 
+## TLS and basic authentication
+
+The Pushgateway supports TLS and basic authentication. This enables better
+control of the API's.
```

Comment:
If I may continue my nitpicking: API's → APIs.

However, the `/metrics` endpoint is not really an API, is it? How about "This enables better control of the various HTTP endpoints."